group_vars/all

```diff
@@ -56,4 +56,8 @@ build_version: v1.20.0-alpha.0.1402+d39214ade1d60c
 # build_version: d39214ade1d60c
 
 ##### Calico Configurations #####
-calico_version: v3.24.1
+calico_version: v3.24.5
+pod_subnet: 172.20.0.0/16
+# The available methods of installation for Calico can be either using the tigera operator, or using the manifest.
+# Choose between "operator" (default) or "manifest" for the desired installation method.
+installation_type: operator
```

Comment:
- lets prefix with calico
- lets have manifest as a default for now, will migrate to operator later

Reply:
Addressed.